g/grpc/grpc_ubuntu_18.04.sh

```diff
@@ -0,0 +1,93 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : GRPC
+# Version       : '1.33.2'
+# Source repo   : https://github.com/grpc/grpc
+# Tested on     : Ubuntu 18.04
+# Script License: MIT License
+# Maintainer    : Lysanne Fernandes <@us.ibm.com>
```

Comment:
@lysannef please correct the email address

Reply:
@seth-priya  corrected email